components/stimulation/StimulationStudioWaveformSettingModal.vue

```diff
@@ -678,8 +693,11 @@ export default {
         this.err_msgs[label] = this.invalid_err_msg.required;
       } else if (isNaN(value)) {
         this.err_msgs[label] = this.invalid_err_msg.num_err;
-      } else if (this.stimulation_type.includes("C") && Math.abs(value) > 100) {
-        this.err_msgs[label] = this.invalid_err_msg.max_current;
+      } else if (
+        this.stimulation_type.includes("C") &&
```

Comment:
Would this become and issue if we have stimulation types that have overlapping letters?

Reply:
Yeah, but we currently don't even have voltage as an option. We can tackle that if it comes to that.